README.md

```diff
@@ -1,23 +1,35 @@
 # DBGenerator
 
-Cet outil permet la génération de modèle de base de données pour les plateformes Android et iOS à partir d'un fichier xcdatamodel. Les SGBD supportés pour le moment sont les suivants :
+DBGenerator is a tool that generates Realm.io classes, for both Android and iOS, from a `.xcdatamodel` file.
+<!--Cet outil permet la génération de modèle de base de données pour les plateformes Android et iOS à partir d'un fichier xcdatamodel. Les SGBD supportés pour le moment sont les suivants :-->

```

Comment:
Reviewing the rest of your translation, I can confirm that it actually makes the review process quite harder if you keep the original translation inside `<!-- … -->` as we then only see _added_ text in the diff then, rather than seing _changes_ (deleted text and what it has been replaced with), so we can't clearly see in the diff what you translated, compared to when GitHub highlights the changes in red (deletion) and green (addition) as we only see green.

_Also the diff count (number of lines added vs. number of lines removed) is misleading if you don't delete the original text and only do additive changes like that_